mol-loader/Chembl.py

```diff
@@ -0,0 +1,137 @@
+"""
+Encapsulate interaction with ChEMBLdb.
+"""
+
+import itertools
+import psycopg2
+
+class Chembl:
```

Comment:
It looks like this never generates an RDKit canonical SMILES. I believe that's necessary.